README.md

```diff
@@ -42,7 +52,36 @@ $ manticore ./path/to/binary ++ ++  # use two symbolic strings of length two as
 
 ### API
 
-Manticore has a Python programming interface which can be used to implement custom analyses.
+Manticore has a Python programming interface which can be used to implement custom analyses and is capable of detailed verification of arbitrary properties of smart contracts via its Python API. Set starting conditions, identify symbolic transactions, then review discovered states to ensure invariants for your contract hold.
```

Comment:
this is a runon sentence, and also has redundancy w.r.t "Python programming interface" / "Python API". i'd like to keep "Manticore has a Python programming interface which can be used to implement custom analyses." as a standalone platform-agnostic first sentence of the intro

maybe something like this?

```
Manticore has a Python programming interface which can be used to implement custom analyses.	

For Ethereum smart contracts, it can be used for detailed verification of arbitrary contract properties. Set starting conditions, *execute* symbolic transactions, then review discovered states to ensure invariants for your contract hold.
```

note: changed identify to execute since i don't understand what "identify symbolic transactions" means.